### ISSUE TITLE
[JOSS review] Compiled documentation

### ISSUE THREAD

**Issue by nick-gorman** [Mat-dp/mat-dp-core#24]
Is the documentation compiled into a pdf or html version anywhere? It's a bit hard to follow reading through the docs folder.

https://github.com/openjournals/joss-reviews/issues/4460

**Comment by kcerva** [comment]
Thank you for this observation @nick-gorman. We've added information on the ReadMe on how to see the documentation- either on mkdocs or with a link to a static site.
